sdk/src/test/java/io/opentelemetry/sdk/metrics/LongSumAggregatorTest.java

```diff
@@ -26,6 +26,6 @@
   @Test
   public void longSumAggregation() {
     LongSumAggregator longSumAggregator = new LongSumAggregator();
-    longSumAggregator.update(12);
+    longSumAggregator.recordLong(12);
   }
 }
```

Comment:
Could you please add also a test class for `DoubleSumAggregator`?

Reply:
Added, so far this is a trivial test until I defined the entire Aggregation class.